lib/filewatcher.rb

```diff
@@ -119,7 +119,7 @@ def filesystem_updated?(snapshot_to_use = nil)
 
     forward_changes.each do |file, mtime|
       @updated_file = file
-      @event = @last_snapshot.fetch(@updated_file, false) ? :changed : :new
```

Comment:
I've been feeling bad about having chosen the event names changed, delete and new. They're inconsistent. 

Since this changes will break existing users code if they update, we'll have to update the gem version number from 0.5.4 to 1.0.0.